rules/windows/process_creation/win_susp_script_exec_from_temp.yml

```diff
@@ -0,0 +1,34 @@
+title: Suspicious Script Execution From Temp Folder
+id: a6a39bdb-935c-4f0a-ab77-35f4bbf44d33
+description: Detects a suspicious script executions from temporary folder
+status: experimental
+references:
+    - https://www.microsoft.com/security/blog/2021/07/13/microsoft-discovers-threat-actor-targeting-solarwinds-serv-u-software-with-0-day-exploit/
+author: Florian Roth, Max Altgelt
+date: 2021/07/14
+tags:
+    - attack.execution
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        Image|endswith:
+            - '\powershell.exe'
+            - '\mshta.exe'
+            - '\wscript.exe'
+            - '\cscript.exe'
+        CommandLine|contains: 
+            - '\Windows\Temp'
+            - '\Temporary Internet'
+            - '\AppData\Local\Temp'
+            - '\AppData\Roaming\Temp'
+            - '%TEMP%'
+            - '%TMP%'
+            - '%LocalAppData%\Temp'
```

Comment:
Why not '\Temp' as it is a pattern ?

Reply:
To avoid false positives in Temp folders of any program sub folder, e.g. C:\ProgramData\Bloomberg\Installer\Temp

Reply:
Ok thanks